package.json

```diff
@@ -15,6 +15,8 @@
     "build:server": "http-server dist",
     "dev": "nx run core:dev",
     "dev:server": "http-server build",
+    "pree2e": "npx playwright install",
+    "e2e": "PORT=5005 MODE=REPLAY npx playwright test",
```

Comment:
`e2e` command is for devs to run e2e tests locally, hence I put `PORT`, `MODE` and `npx playwright install` here.